### ISSUE TITLE
Airplay 2?

### ISSUE THREAD

**Issue by AppelBoomHD** [philippe44/ShairTunes2#53]
Would it be possible to implement Airplay 2 by using one of the newer versions of shairport-sync (https://github.com/mikebrady/shairport-sync) that support it?

**Comment by philippe44** [comment]
no, not in my plans. Too complicated and no interest for me, sorry
